dashboard/app/views/home/_homepage.html.haml

```diff
@@ -13,3 +13,21 @@
 .clear{style: "clear: both"}
 
 #homepage-container
+
+#admin-extras
+  -if current_user.admin?
+    %a{href: CDO.code_org_url('/ops-dashboard#/districts'), style: 'text-decoration: none;'}
+      %h2
```

Comment:
It feels slightly odd to use a heading as a link style.  Is it worth making a new style for these links?